server/repository/migrations/sqlite/2022-04-27T13-05_create_document_trigger/down.sql

```diff
@@ -0,0 +1,3 @@
+DROP TRIGGER IF EXISTS document_insert_trigger;
+DROP TRIGGER IF EXISTS document_update_trigger;
+DROP TRIGGER IF EXISTS document_delete_trigger;
```

Comment:
Was wondering, when is document deletion allowed ? Can document be referenced by something that is not deleted, and when deleted causing errors somewhere (potentially of-site) -> runtime for constraint or logic if no constraints

There should really be a rule for only soft deleting central or `shared` data.

Reply:
Also, can document be updated ?